src/sentry/models/organization.py

```diff
@@ -511,11 +511,13 @@ def handle_email_verification_required(self, request):
                 request, remove_email_verification_non_compliant_members
             )
 
-    def get_url_viewname(self):
+    @staticmethod
+    def get_url_viewname():
         return "sentry-organization-issue-list"
 
-    def get_url(self):
-        return reverse(self.get_url_viewname(), args=[self.slug])
+    @staticmethod
+    def get_url(slug: str):
+        return reverse(Organization.get_url_viewname(), args=[slug])
```

Comment:
More extracting this logic into static methods so that control silo can use them without having ORM object in hand.